en/includes/guides/organization-management/email-domain-based-organization-discovery.md

```diff
@@ -27,9 +27,25 @@ and user Bob and Ben, with emails `bob@gcmart.com` and `ben@glory.com` should be
 ### Using the Console
 
 1. Login to the organization (root) from the {{ product_name }} Console.
-2. On the {{ product_name }} Console, go to **Login & Registration**, and click **Email Domain Discovery** under **Organization Settings**.
+2. On the {{ product_name }} Console, go to **Login & Registration**, and click **Organization Discovery** under **Organization Settings**.
 3. Turn on the toggle to enable email domain based organization discovery.
 
+    !!! note
+        When this is enabled, following restrctions will apply to child organizations during federated authentication and user onboarding.
+
+        - Users can self-register, and administrators can onboard users to child organizations, only if the users' email domains match the domains mapped to the corresponding child organization.
+
+        {% if (product_name == "WSO2 Identity Server") %}
+
+        - If the Identity Provider (IDP) includes a mapped email attribute, federated authentication and Just-In-Time (JIT) provisioning for child organizations are restricted to email domains mapped to those child organizations.
+
+        {% endif %}
+{% if (product_name == "WSO2 Identity Server") %}
+
+4. Select the **Email domain discovery for self-registration** checkbox if you want to allow users to discover and self-register in child organizations based on their email domain.
+
```

Comment:
Mention the required deployment.toml config to enable self signup in every sub org level, and mention the current limitation of self registration controlling governance configs are not inheriting from org hierarchy